### PR TITLE
prevent CA cert updater from running on forks

### PR DESCRIPTION
**Pull request by gdams**
I can't be the only one getting these emails frequently. This PR prevents this

Screenshot 2022-07-06 at 09 02 02

### CODE REVIEW

**Comment by zdtsw** [comment]
i simply disabled GH action in my git repo, peace forever

**Review by zdtsw — APPROVED** [review]
LGTM

**Comment by gdams** [comment]
@zdtsw that seems strange... the [election](https://projects.eclipse.org/projects/adoptium.temurin/elections/election-wen-zhou-committer-eclipse-temurin%E2%84%A2) was completed some time ago (I'm pretty sure it's an automated process from that point)

File: .github/workflows/ca-cert-updater.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   update_ca_certs:
+    if: startsWith(github.repository, 'adoptium/')
```
Comment (tellison) [inline .github/workflows/ca-cert-updater.yml:10]: That works, but could also use `github.repository_owner == 'adoptium'

**Pull request merged**